### ISSUE TITLE
Share Accounts not sorted

### ISSUE THREAD

**Issue by spillaip** [moneymanagerex/moneymanagerex#6739]
MMEX version:
 - [ ] 1.7.0
 - [X] 1.8.0
 - [ ] 1.8.1
 - [ ] 1.8.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Please describe your issue with details.
Share accounts are listed as entered, need to be listed alphabetically

Add screenshot or other files if needed.
![image](https://github.com/moneymanagerex/moneymanagerex/assets/13807297/4bd03137-1ed8-4b06-b01a-fb9b47926bad)

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by spillaip** [comment]
@n-stein  Your observation is correct, I tried to edit the account and try to remove the leading space, but getting error:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/13807297/d33c34c4-c334-40e4-9dad-8d0a6f03a553)

**Comment by n-stein** [comment]
Do you have the same account name with and without the space? The system won't let you duplicate account names.

If that is the case, go into the account with the space and Ctrl+A to select all transactions, then right click and select "Move Transactions" and pick the account without the space. Then just delete the empty account with the space.

**Comment by spillaip** [comment]
Yes, there is a similar account with leading space.  I have pressed Ctrl+A and the right click providing me with

![image](https://github.com/moneymanagerex/moneymanagerex/assets/13807297/0cea09b3-451c-4bc4-ab51-7e3d9184ad4e)
